Modules/Devices/src/Devices.Domain/Entities/Identities/Identity.cs

```diff
@@ -115,7 +115,7 @@ public IdentityDeletionProcess StartDeletionProcessAsOwner(DeviceId asDevice)
         DeletionGracePeriodEndsAt = deletionProcess.GracePeriodEndsAt;
         TierId = Tier.QUEUED_FOR_DELETION.Id;
         Status = IdentityStatus.ToBeDeleted;
-        RaiseDomainEvent(new IdentityToBeDeletedDomainEvent(Address, deletionProcess.GracePeriodEndsAt));
+        RaiseDomainEvent(new IdentityToBeDeletedDomainEvent(Address, (DateTime)deletionProcess.GracePeriodEndsAt!));
```

Comment:
This looks a bit wierd (though I understand that it somehow has to be converted, probably bc it's a property instead of a variable). My solution would be

```csharp
RaiseDomainEvent(new IdentityToBeDeletedDomainEvent(Address, deletionProcess.GracePeriodEndsAt!.Value));
```

But I don't know if there's a convention for this kind of situation, so I would leave that to @tnotheis to decide.

Reply:
I would have done it just like you suggested it 👍